### PR TITLE
Build Ubuntu 20.04 CI images and containers

### PR DESCRIPTION
**Pull request by CyanDevs**
This PR is downstream of the following PR:
* https://github.com/openenclave/openenclave/pull/3864

This PR depends on the following to be merged first:
* https://github.com/openenclave/openenclave/pull/3829
* https://github.com/openenclave/openenclave-ci/pull/28

This also uses a docker image updated from Ansible 2.8.8 to 2.9.18 as a requirement to building Ubuntu 20.04 due to [Ansible issue #68536](https://github.com/ansible/ansible/issues/68536). From the upgrade [Ansible issue #71634](https://github.com/ansible/ansible/issues/71634) occurred so this PR also fixes importing GPG keys for yum repos on RHEL8.

Notes:
* Azure DCAP package for Ubuntu 20.04 is not available yet so it is skipped in Ansible. This will be restored when the package is available.
* This does not include running tests as part of our CI yet as we are not ready to begin testing on Ubuntu 20.04 due to the above.

### CODE REVIEW

File: scripts/ansible/oe-linux-acc-setup.yml [hunk 1]
```diff
@@ -22,6 +22,7 @@
     - import_role:
         name: linux/az-dcap-client
         tasks_from: stable-install.yml
+      when: ansible_distribution_version != "20.04"
```
Comment (rs--) [inline scripts/ansible/oe-linux-acc-setup.yml:25]: Is this because the `az-dcap-client` isn't available for 20.04 yet?

Comment (CyanDevs) [reply scripts/ansible/oe-linux-acc-setup.yml:25]: Yes, that is correct. This line is to be removed when the package is available so the final production-ready image can be created.

**Comment by CyanDevs** [comment]
Cherry-picked commit from https://github.com/openenclave/openenclave/pull/3829/commits to merge our work so that bors can build successfully

**Review by rs-- — APPROVED** [review]

**Review by radhikaj — APPROVED** [review]

**Comment by anakrish** [comment]
Seems to have failed in submodule update.

@CyanDevs One option to consider is `git submodule update --init --recursive --depth 1 --recommend-shallow` . This does a shallow update of the submodule.

**Comment by CyanDevs** [comment]
Thanks for the suggestion @anakrish I'll try it out!

**Pull request merged**